index-cn.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf8">
+<title>活动守则</title>
+<style>
+* {
+  font-family: "Microsoft YaHei",  "微软雅黑体", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: 200;
+}
+
+html {
+  background: #efefef;
+}
+
+body {
+  background: white;
+  color: #212121;
+  margin: 0;
+  padding-top: 60px;
+}
+
+p, h1, h2 {
+  max-width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+h1 {
+  font-size: 38px;
+}
+
+h2 {
+  font-size: 28px;
+  margin-top: 48px;
+}
+
+p {
+  line-height: 27px;
+  font-size: 18px;
+  text-align: justify;
+}
+
+a {
+  color: blue;
+}
+
+em {
+  font-weight: 400;
```

Comment:
Override the user agent stylesheet to enforce non-italic presentation here. zh text should not be presented in oblique. See replacement at https://zh.wikipedia.org/wiki/Template:Zh-em.